backend/database/models/officer.py

```diff
@@ -0,0 +1,38 @@
+from .. import db
+from .types.enums import Gender
+from .types.enums import Race
+import enum
+
+
+class Rank(str, enum.Enum):
+    # TODO: Is this comprehensive?
+    TECHNICIAN = "TECHNICIAN"
+    OFFICER = "OFFICER"
+    DETECTIVE = "DETECTIVE"
+    CORPORAL = "CORPORAL"
+    SERGEANT = "SERGEANT"
+    LIEUTENANT = "LIEUTENANT"
+    CAPTAIN = "CAPTAIN"
+    DEPUTY = "DEPUTY"
+    CHIEF = "CHIEF"
+
+
+class Officer(db.Model):
+    id = db.Column(db.Integer, primary_key=True)  # officer id
+    first_name = db.Column(db.Text)
+    last_name = db.Column(db.Text)
+    race = db.Column(db.Enum(Race))
+    gender = db.Column(db.Enum(Gender))
+    appointed_date = db.Column(db.DateTime)
+    badge = db.Column(db.Text)
+    unit = db.Column(db.Text)  # type?
+    # TODO: How do we handle changes over time in rank?
```

Comment:
Each officer object will be related to a single event; I think we should be fine just capturing instantaneous rank in that case. For a combined officer object we can take the most recent rank.